client/lib/plans/constants.js

```diff
@@ -263,7 +263,6 @@ export const PLANS_LIST = {
 			FEATURE_WORDADS_INSTANT,
 			FEATURE_VIDEO_UPLOADS_JETPACK_PREMIUM,
 			FEATURE_MALWARE_SCANNING_DAILY,
-			FEATURE_UNLIMITED_PREMIUM_THEMES,
```

Comment:
I might be wrong, but doesn't the `hasFeature` check [rely](https://github.com/Automattic/wp-calypso/blob/d2dd377136ffb602d7d657c422934761e6f8bbdd/client/state/sites/plans/selectors.js#L216-L218) on the presence of this?